### DISCUSSION TITLE
sloeber/eclipse pre-build step make (e=5): Access is denied.

### DISCUSSION THREAD

**Discussion by PerennialNovice** [stm32duino/Arduino_Core_STM32#2295] [Q&A]
Hi,

I have a little batch script I execute before building that fetches a git commit-hash to be added as constexpr to the source code.
This works fine for AVR targets, but when I try to use the very sam batch file for STM32 I get an error

C:\Users\USER\Documents\PROJECT\fetchCommitHash.bat
process_begin: CreateProcess(C:\Users\USER\Documents\PROJECT\fetchCommitHash.bat, C:\Users\USER\Documents\PROJECT\fetchCommitHash.bat, ...) failed.
make (e=5): Access is denied.

I use latest STM32 core 2.7.1 and sloeber 4.43

Any idea what might be causing this error?
the script works on the command line, but not when called as a pre-build step from within sloeber

**Comment by fpistm** [comment]
Same answer:
https://github.com/orgs/stm32duino/discussions/2242#discussioncomment-8021841

Here we support only Arduino IDE. You can try on the forum.
